set.mm

```diff
@@ -281762,6 +281845,115 @@ a tuple of a topological space (a member of ` TopSp ` , not ` Top ` )
       CVWHUVAUVBUPUVCUVDUVEUVFUVGUVHUVI $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                         Complex vector spaces
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c CVec $. $( Complex vector space $)
+
+  $( Complex vector space. $)
+  ccvs $a class CVec $.
+
+  $( Define a complex vector space, which is just a complext left module and a
```

Comment:
Nit.. complext should be complex.

Reply:
This will be fixed with the next commit!

Reply:
> This will be fixed with the next commit!

I  fixed complext -> complex in 57b7d75